talisker/__init__.py

```diff
@@ -148,6 +149,69 @@ def run():
         sys.exit(1)
 
 
+def format_docstring(docstring, width):
+    short, _, long = docstring.partition('\n\n')
+    short = textwrap.wrap(' '.join(short.split()), width)
+    if long:
+        long = textwrap.wrap(' '.join(long.split()), width)
+    return short, long
+
+
+def run_help():
+    """
+    Usage: talisker.help [CONFIG NAME]
+
+    Talisker provides some executable wrappers, which initialise Talisker
+    and then simply pass through any supplied arguments to underlying command.
+
+     - talisker.gunicorn wraps the regular gunicorn invocation
+     - talisker.celery wraps the celery command
+     - talisker.run wraps a regular call to python, and takes a script to run
+
+    Talisker can be configured by the environment variables listed below. These
+    variable can also be supplied in a python file, although environment
+    variables override any file configuration.
+    """
+    width = 80
+    indent = 30
+    rest = width - indent
+    indent_str = '\n' + ' ' * indent
+    metadata = get_config().metadata()
+
+    if len(sys.argv) > 1:
+        name = sys.argv[1]
+        if name.upper() not in metadata:
+            sys.stderr.write('Invalid config: {}\n'.format(name))
+            sys.exit(1)
```

Comment:
You can do this all in one with 
```suggestion
            sys.exit('Invalid config: {}'.format(name))
```